commands/cluster-addslotsrange.md

```diff
@@ -0,0 +1,51 @@
+This command is useful in order to modify a node's view of the cluster
```

Comment:
We typically don't duplicate the information between two similar commands. I would include the syntax information here, and then a link to the `ADDSLOTS` subargument.

Reply:
I don't think this was addressed. I think we can simplify this page to just talk about how it's different from cluster addslots, since I don't want to duplicate so much context.